contentcuration/contentcuration/frontend/settings/pages/UsingStudio/index.vue

```diff
@@ -118,25 +116,21 @@
       bestPractice2:
         'It is preferable to create multiple small channels rather than one giant channel with many layers of folders.',
       bestPractice3:
-        'Reload the page often to ensure your work is saved to the server and no network errors have occurred. Use CTRL+R on Linux/Windows or ⌘+R on Mac.',
-      bestPractice4:
-        'Avoid simultaneous edits on the same channel. Channels should not be edited by multiple users at the same time or by the same user in multiple browser windows.',
+        'Reloading the page should help ensure your work is saved to the server. Use CTRL+R on Linux/Windows or ⌘+R on Mac.',
```

Comment:
I suggest keeping the 'Reload' here instead of the _-ing_ form: it's more straightforward, and in line with phrasing of other recommendations (Wait..., Compress..., Publish...)

`Reload the page to confirm your work has been saved on the server. Use CTRL+R on Linux/Windows or ⌘+R on Mac.`

Reply:
Done!